lib/Driver/DarwinToolChains.cpp

```diff
@@ -583,3 +585,16 @@ bool toolchains::Darwin::shouldStoreInvocationInDebugInfo() const {
     return S[0] != '\0';
   return false;
 }
+
+void toolchains::Darwin::validateArguments(DiagnosticEngine &diags, const llvm::opt::ArgList &args) const {
+    
+    // Validating darwin unsupported -static-stdlib argument.
+    if (const Arg *A = args.getLastArg(options::OPT_target)) {
+        auto TargetTriple = llvm::Triple::normalize(A->getValue());
+        const llvm::Triple target(TargetTriple);
```

Comment:
Looks like you no longer need this part at all, since we already know it's a Darwin triple. (Also, two-space indents, please!)

Reply:
That's true fixing :))